Include/internal/pycore_compile.h

```diff
@@ -72,8 +182,13 @@ int _PyCompile_EnsureArrayLargeEnough(
         int default_alloc,
         size_t item_size);
 
-int _PyCompile_ConstCacheMergeOne(PyObject *const_cache, PyObject **obj);
+int _PyCompiler_ConstCacheMergeOne(PyObject *const_cache, PyObject **obj);
+
+PyCodeObject *_PyCompile_OptimizeAndAssemble(struct compiler *c, int addNone);
 
+Py_ssize_t _PyCompile_DictAddObj(PyObject *dict, PyObject *o);
+int _PyCompiler_Error(struct compiler *c, _Py_SourceLocation loc, const char *format, ...);
```

Comment:
Why do some function have a Compile prefix and others have Compiler?

Reply:
It used to be `_PyCompile_XXX`, and these were functions called from outside of the compilation machinery to invoke it. The new API is for codegen, for when it needs to perform operations on the Compiler struct. So I named those `_PyCompiler_`. 

`_PyCompile_OptimizeAndAssemble` takes a compiler, but it's not performing operation on it (it's consuming it to produce a code object) so I wasn't sure it should get a `_PyCompiler_` prefix.

Reply:
`_PyCompiler_ConstCacheMergeOne` should be `_PyCompile_ConstCacheMergeOne`.

Reply:
for everything?

Reply:
Yes, that's my suggestion (though I'd leave the existing `PyCompilerFlags` alone).

Reply:
Yeah, ok. It will be simpler.